packages/web/src/pages/rewards-page/components/ClaimAllPanel.tsx

```diff
@@ -0,0 +1,97 @@
+import { useCallback } from 'react'
+
+import {
+  formatCooldownChallenges,
+  useChallengeCooldownSchedule
+} from '@audius/common/hooks'
+import { Flex, IconTokenGold, Paper, Text } from '@audius/harmony'
+
+import { useModalState } from 'common/hooks/useModalState'
+import { useIsMobile } from 'hooks/useIsMobile'
+
+import { messages } from '../messages'
+
+export const ClaimAllPanel = () => {
+  const isMobile = useIsMobile() || window.innerWidth < 1080
```

Comment:
what is this? scared of 1080 hardcoded value + is this a new pattern we should adopt?

Reply:
i just split some components that were in a single file out to their own files for better readability, this has existed in the codebase for sometime